becquerel/core/spectrum.py

```diff
@@ -341,24 +353,73 @@ def cpskev_vals(self):
 
     @property
     def cpskev_uncs(self):
-        """Uncertainties on the counts per second per keV in each channel.
+        """Uncertainties on the counts per second per keV in each bin.
 
         Returns:
           an np.array of floats
         """
 
         return unumpy.std_devs(self.cpskev)
 
+    @property
+    def bins(self):
+        """Bin indices.
+
+        Returns:
+          np.array of int's from 0 to (len(self.counts) - 1)
+        """
+
+        return np.arange(len(self), dtype=int)
+
     @property
     def channels(self):
-        """Channel index.
+        """Bin indices.
 
         Returns:
           np.array of int's from 0 to (len(self.counts) - 1)
```

Comment:
This is an alias for `self.bins`. Add that this warns and will be removed in a future version.